src/ts/components/settingspanelitem.ts

```diff
@@ -11,47 +11,74 @@ import {PlaybackSpeedSelectBox} from './playbackspeedselectbox';
 import { PlayerAPI } from 'bitmovin-player';
 import { LocalizableText } from '../localization/i18n';
 
+/**
+ * Configuration interface for a {@link SettingsPanelItem}
+ *
+ * @category Configs
+ */
+export interface SettingsPanelItemConfig extends ContainerConfig {
+  /**
+   * The label component or the text for the label.
+   */
+  label?: LocalizableText | Component<ComponentConfig>;
+  /**
+   * The component that configures a setting.
+   */
+  setting?: Component<ComponentConfig>;
```

Comment:
Nit: Maybe it should just be called component.

Reply:
I went for `settingComponent` 👍 [59a7bf5](https://github.com/bitmovin/bitmovin-player-ui/pull/655/commits/59a7bf5ec18e6c9b2615dbb7240dd53fe8dcaac1)